sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/Document.java

```diff
@@ -31,40 +31,67 @@ public Document(T document, String partition, String id) {
         this.document = document;
     }
 
+    /**
+     * Create document from error.
+     * @param dError General exception.
+     */
     public Document(Exception dError) {
         this.documentError = new DocumentError(dError);
     }
 
     /**
-     * @return Deserialized document (or null)
+     * Get document.
+     * @return Deserialized document (or null).
```

Comment:
When is it null?

Reply:
:man_facepalming: just used the original comment let me fix that....